configure

```diff
@@ -170,6 +171,7 @@ print_feature()
 printf "...%15s: %s\n" "prefix" "${prefix}"
 print_feature "libelf" "have_libelf" "more flexible ELF data handling"
 print_feature "libdw" "have_libdw" "DWARF debug info support"
+print_feature "libcapstone" "have_libcapstone" "Disassembler to full-dynamic"
```

Comment:
Please put this in the last line with message as "full dynamic tracing support".